app/jobs/document_proofing_job.rb

```diff
@@ -12,8 +12,7 @@ def perform(
     trace_id:,
     image_metadata:,
     analytics_data:,
-    flow_path:,
-    liveness_checking_enabled: nil
```

Comment:
Speaking of being careful with args, we need to leave this one in since this is a background job

Reply:
Put back.